inc/templates/pattern-library.php

```diff
@@ -15,32 +15,81 @@
 
 \get_header();
 
+?>
+
+<style>
+	.kalutara-component {
+		border-bottom: 2px solid #333 !important;
+		padding: 0 10px 10px !important;
+		background: white !important;
+	}
+	.kalutara-component:last-child {
+		border-bottom: none !important;
+	}
+	.kalutara-component__header {
+		background: #EEE !important;
+		padding: 5px 10px !important;
+		margin: 0 -10px 10px !important;
+		overflow: hidden !important;
+	}
+	.kalutara-component__summary,
+	.kalutara-component__template,
+	.kalutara-component__documentation,
+	.kalutara-component__variant-title {
+		margin: 5px 0 !important;
+		padding: 0 !important;
+		font-size: 14px !important;
+		line-height: 1.4 !important;
+		font-family: "Menlo", monospace !important;
```

Comment:
Maybe use a full font stack here rather than just an OSX font? This is GitHub's font stack:
```suggestion
		font-family: SFMono-Regular, Consolas, Liberation Mono, Menlo, monospace !important;
```

Reply:
Good call. I've updated this.